thread1:
  core: 1
thread2:
  core: 1
thread3:
  core: 2
thread4:
  core: 3
thread5:
  core: 4
thread6:
  core: 4
